File: linear_search/python/linear_search.py
def linear_search(array , target):
    for index in range(len(array)):
        if array[index] == target:
            return index

    return -1


array = [23,45,1,2,3,6,9,1,10]
target = 9
ans = linear_search(array,target)
print(ans)

File: linear_search/java/linear_search.java
package linear_search.java;
public class Main{
    public static void main(String[] args){
        int[] nums = {23,45,1,2,3,6,9,1,10};
        int target = 9;
        int ans = linear_search(nums, target);
        System.out.println(ans);
    }

    // search in the array: return index if item found else -a
    static int linear_search(int[] arr, int target){
        for (int index=0; index<arr.length; index++){
            if (target == arr[index]){
                return index;
            }
        }
        return -1;
    }
}